policy/modules/apps/gpg.if

```diff
@@ -435,3 +544,22 @@ interface(`gpg_list_user_secrets',`
 	list_dirs_pattern($1, gpg_secret_t, gpg_secret_t)
 	userdom_search_user_home_dirs($1)
 ')
+
+########################################
+## <summary>
+##	Do not audit attempt to search gpg user secrets dirs.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
```

Comment:
This interface is not allowing access.

Reply:
What do you mean ?

It simply prevents auditing. It does not grant nor deny any permimssion.

Reply:
I suppose you mean the comment... I'll amend that.